google/cloud/aiplatform/metadata/schema/google_schema.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this module should be private as only Vertex(whether service, sdk, or pipeline component) should create these metadata resources.  

@jialuzh @IronPan

Reply:
Good point, @IronPan  correct me if i'm wrong here though I believe FPC is currently using similar methods from types in their YAML definition see [types definition](https://github.com/kubeflow/pipelines/blob/5baa4879a9fe33315f3f33338dff418dc2abbf69/components/google-cloud/google_cloud_pipeline_components/types/artifact_types.py) and [use in yaml](https://github.com/kubeflow/pipelines/blob/5baa4879a9fe33315f3f33338dff418dc2abbf69/components/google-cloud/google_cloud_pipeline_components/v1/dataset/export_text_dataset/component.yaml). This could be a reason to keep the method public.

Reply:
resolving this issue as wont fix. kindly let me know if any follow up is needed.